bin/extract.sh

```diff
@@ -10,6 +10,16 @@ do
     echo "...  $name"
     cp $corpus txt/$name.txt
 done
+sed -i '/\[Illustration\]/d'  txt/*.txt
+sed -i '1,/START/d' txt/*.txt
+sed -i '/END OF THE PROJECT GUTENBERG/,$d' txt/*.txt
+sed -i '/End of Project/d' txt/*.txt
```

Comment:
```suggestion
sed -i '/End of Project Gutenberg/d' txt/*.txt
```

Reply:
I’d prefer to be as strict as possible for these start/end markers. Maybe making sure they appear at the beginning of the line (with a `^` mark) would be a good addition.